hack/local-up-cluster.sh

```diff
@@ -476,6 +469,11 @@ EOF
       --feature-gates="${FEATURE_GATES}" \
       --cloud-provider="${CLOUD_PROVIDER}" \
       --cloud-config="${CLOUD_CONFIG}" \
+      --requestheader-username-headers=X-Remote-User \
```

Comment:
@sttts any reason not to do this.  It makes us compatible with `kubernetes-discovery`.